utils/Formatter.tsx

```diff
@@ -225,22 +201,20 @@ export function formatAsCoins(number: number): string {
     return `${numberWithThousandsSeparators(number)} Coins`
 }
 export function formatDungeonStarsInString(stringWithStars: string, style: CSSProperties = {}, dungeonItemLevelString?: string): JSX.Element {
-    let yellowStarStyle = { color: 'yellow', fontWeight: 'normal', height: '100%' }
+    let yellowStarStyle = { color: '#ffaa00', fontWeight: 'normal', height: '100%' }
     let redStarStyle = { color: 'red', fontWeight: 'normal', height: '100%' }
     let itemNameStyle = {
         height: '32px',
         marginRight: '-5px'
     }
     let stars = stringWithStars?.match(/✪.*/gm)
 
-    let numberOfMasterstars = undefined
+    let numberOfMasterstars = 0;
     if (dungeonItemLevelString) {
         try {
-            let number = parseInt(dungeonItemLevelString)
-            if (number > 5) {
-                numberOfMasterstars = number - 5
-            }
-        } catch {}
+            
+            numberOfMasterstars = Math.max(parseInt(dungeonItemLevelString) - 5, 0)
+        } catch { }
```

Comment:
Remove the ";"
And the empty line after try {

This shoud both automatically be fixed by the formatter. I recently pushed a prettierrc config file that should be used